crates/extensions/tedge_http_ext/src/tests.rs

```diff
@@ -15,8 +15,8 @@ async fn get_over_https() {
 }
 
 async fn spawn_http_actor() -> ClientMessageBox<HttpRequest, HttpResult> {
-    let mut builder = HttpActorBuilder::new().unwrap();
-    let handle = ClientMessageBox::new("Tester", &mut builder.box_builder);
+    let mut builder = HttpActor::new().builder();
```

Comment:
That looks very unusual, creating a builder out of a concrete instance, as builders are typically used to build the instance. But here, it's the reverse. Something like `HttpActor::builder()` would have been more natural.

Reply:
Agree - even if I don't read the same way as you (getting a builder from an instance). Will see why I did that and how this can be changed.